psuutil/main.py

```diff
@@ -84,7 +84,7 @@ def numpsus():
 @click.option('-i', '--index', default=-1, type=int, help='Index of the PSU')
 def status(index):
     """Display PSU status"""
-    header = ['PSU',  'Model', 'Serial', 'Voltage (V)', 'Current (A)', 'Power (W)', 'Status', 'LED']
+    header = ['PSU',  'Model', 'Serial', 'HW Rev', 'Voltage (V)', 'Current (A)', 'Power (W)', 'Power Warn-supp Thres (W)', 'Power Crit Thres (W)', 'Status', 'LED']
```

Comment:
@stephenxs could you please run sonic-mgmt test with all of your changes and attach the test result please

Reply:
Sure. I do not have a testbed now and will do it next Monday.